routing/endpointregistry.go

```diff
@@ -31,6 +34,10 @@ func (e *entry) InflightRequests() int64 {
 	return e.inflightRequests
 }
 
+func (e *entry) FadeInDuration() time.Duration {
```

Comment:
Do we need getter for exponent as well?

Reply:
I can add it in the current PR, but we will need it only in the next PR (when loadbalancer algorithm will use it), so I decided to postpone this getter to the next PR.